components/Slider/examples/SliderCollectionViewController.m

```diff
@@ -16,9 +16,9 @@
 
 #import "MaterialCollections.h"
 #import "MaterialColorScheme.h"
+#import "MaterialTypographyScheme.h"
```

Comment:
Nit/order of imports:

0. Related header (if one exists) to declare the interface
1. UIKit/system
2. Alphabetized MDC imports.

Reply:
Done